docs/production-deployment/cloud/account-setup/users.mdx

```diff
@@ -122,13 +148,9 @@ For a Namespace, a user can have one of the following permissions:
 
 ## How to update an account-level Role in Temporal Cloud {#update-roles}
 
-You can update the account-level [Role](#account-level-roles) for a user by using either Web UI or tcld.
-
-:::info
-
-To update an account-level Role, a user must have the Global Admin account-level Role.
-
-:::
+With Global Admin or Account Owner privileges, you can update any user's account-level [Role](#account-level-roles) using either the Web UI or `tcld`.
+This does not apply to the Account Owner role.
+To create, update, or delete an Account Owner, you must submit a [support ticket](https://temporalsupport.zendesk.com/).
```

Comment:
suggestion: combine the bullet points on line 152 and 153, maybe something like "For security purposes, Account Owners can only be changed through Temporal Support. To create, update, or delete an Account Owner, you must submit a [support ticket](https://temporalsupport.zendesk.com/)."

Reply:
These aren't bullet points. They're sentences. On the docs team, we put each sentence on one line so we can more easily diff and inspect changes.

Reply:
```
With Global Admin or Account Owner privileges, you can update any user's account-level [Role](#account-level-roles) using either the Web UI or `tcld`.
This does not apply to the Account Owner role.
For security purposes, Account Owners can only be changed through Temporal Support.
To create, update, or delete an Account Owner, you must submit a [support ticket](https://temporalsupport.zendesk.com/).
```